jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/operation/GenerateWebServerRunSteps.java

```diff
@@ -0,0 +1,27 @@
+package com.cerner.jwala.ui.selenium.steps.operation;
+
+import com.cerner.jwala.ui.selenium.component.JwalaUi;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.openqa.selenium.By;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Created by Jedd Cuison on 8/14/2017
+ */
+public class GenerateWebServerRunSteps {
+
+    @Autowired
+    private JwalaUi jwalaUi;
+
+    @When("^I click the \"Generate Web Servers\" button of group \"(.*)\"$")
+    public void clickGenerateWebServersBtnOfGroup(final String groupName) {
+        jwalaUi.click(By.xpath("//tr[td[text()='" + groupName + "']]/following-sibling::tr//button[span[text()='Generate Web Servers']]"));
```

Comment:
I generally prefer using clickWhenReady instead in operations page since the  gears are visible sometime after expanding the group